src/challengePaths/2015/CommunityService.ts

```diff
@@ -312,7 +321,11 @@ export const BoozeDrop = new Test(
     return (
       60 -
       multiplier *
-        Math.floor((getModifier("Item Drop") - familiarItemDrop) / 30 + 0.001) -
+        Math.floor(
+          (getModifier("Item Drop") - familiarItemDrop - familiarBoozeDrop) /
+            30 +
+            0.001
```

Comment:
Can we get a comment above for why we're adding 0.001? Same as below. I know this is not introduced in this PR